c-deps/REPOSITORIES.bzl

```diff
@@ -0,0 +1,44 @@
+# Loading part for c-deps third party repositories.
```

Comment:
nit: let's make this comment `# c-deps repository definitions`

Reply:
Done